web/themes/joinup/prototype/sass/components/_chip.sass

```diff
@@ -0,0 +1,54 @@
+// Chip
+//
+// Markup: todo.
+//
+
+// Component
+// --------------------------------------------------
+.mdl-chip
+  height: 32px
+  line-height: 32px
+  padding: 0 12px
+  border: 0
+  border-radius: 16px
+  background-color: #dedede
+  display: inline-block
+  color: rgba(0,0,0,.87)
+  margin: 2px 0
+  font-size: 0
+  white-space: nowrap
+
+.mdl-chip__text
+  font-size: 13px
+  vertical-align: middle
+  display: inline-block
+
+.mdl-chip__action
+  height: 24px
+  width: 24px
+  border-radius: 12px
+  background: #767676 none !important
+  opacity: .54
+  cursor: pointer
+  padding: 0
+  margin: 0 0 0 4px
+  font-size: 13px
+  text-decoration: none
+  color: rgba(0,0,0,.87)
+  border: none
+  outline: none
+  display: inline-block
+  vertical-align: middle
+  overflow: hidden
+  text-align: center
+
+  &:focus,
+  &:active
+    background-color: #4e4e4e !important
+
+  .icon
```

Comment:
The icons seem to be a bit tilted on the left. If we add here a
```
position: relative;
left: 1px;
```
the issue is solved.

Reply:
As explained in the JIRA ticket the icons are temporary and no time should be spent on it. A related front-end ticket will take care of the styling before release.